content/en/docs/tasks/extend-kubernetes/configure-multiple-schedulers.md

```diff
@@ -37,6 +37,7 @@ and build the source.
 ```shell
 git clone https://github.com/kubernetes/kubernetes.git
 cd kubernetes
+git reset --hard $(kubectl version --short=true | grep "Server Version" | awk '{print $3}')
```

Comment:
@divya-mohan0209  Should the `kubectl` command include the `--short` option?
Thanks

Reply:
@kbhawkey I wouldn't recommend it. See related [issue #1216](https://github.com/kubernetes/kubectl/issues/1216)

Reply:
Hi @chenxinlong
Can you use the command without the `--short=true` option?